CMakeLists.txt

```diff
@@ -295,3 +295,11 @@ if(LINUX)
     )
     install(SCRIPT "${CMAKE_SOURCE_DIR}/cmake/CreateAppImage.cmake")
 endif()
+if (MACOS)
```

Comment:
Not important but you could make a CreateDMG.cmake and do 
`install(SCRIPT "${CMAKE_SOURCE_DIR}/cmake/CreateDMG.cmake")`
like I did for the linux AppImage

Reply:
Ah, right,. I'll clean that up later.